core/src/main/java/com/oath/oak/OakRBuffer.java

```diff
@@ -133,13 +133,4 @@
      * @return The return value of the transform
      */
     <T> T transform(Function<ByteBuffer, T> transformer);
-
```

Comment:
This is unsafe because the user should be aware that the endianness of this data will not be compatible with the buffer endianness as used by, for example, `getInt()`.
This is extremely problematic as the user cannot modify the endianness of the buffer without using the unsafe interface.
Regarding the lock, the user can still use the unsafe interface inside a locked context and implement its own array copy method.